call.go

```diff
@@ -161,8 +205,9 @@ func invoke(ctx context.Context, method string, args, reply interface{}, cc *Cli
 		)
 		// TODO(zhaoq): Need a formal spec of fail-fast.
 		callHdr := &transport.CallHdr{
-			Host:   cc.authority,
-			Method: method,
+			Host:     cc.authority,
+			Method:   method,
+			FailFast: c.failFast,
```

Comment:
can we put fail fast into Begin stats so that it is not needed to pass it into transport?